agimus_controller/agimus_controller/warm_start_reference.py

```diff
@@ -61,14 +64,14 @@ def generate(
 
         x_init = [
             np.hstack([point.robot_configuration, point.robot_velocity])
-            for point in [initial_state] + reference_trajectory
+            for point in itertools.chain([initial_state], reference_trajectory[1:])
```

Comment:
I thought we agreed on x_init being x0 + n_states array. Now the whole x_init is of size n_states? Or do I understand the notion of n_states wrong?

Reply:
`reference_trajectory` is expected to be of size `n_states` as well.

Reply:
okay, so final agreement is that x_init is of size n_states and u_init is of size (n_states-1) and then we create (n_states-1) running models, right?